packages/hoppscotch-common/src/pages/e/_id.vue

```diff
@@ -18,6 +18,41 @@
       :properties="properties"
       :shared-request-i-d="sharedRequestID"
     />
+    <div v-else class="flex flex-1 flex-col items-center justify-center p-4">
+      <div
+        v-if="sharedRequestDetails.loading"
+        class="flex flex-1 flex-col items-center justify-center p-4"
+      >
+        <HoppSmartSpinner />
+      </div>
+      <div v-else>
+        <div
+          v-if="
+            !sharedRequestDetails.loading && E.isLeft(sharedRequestDetails.data)
+          "
+          class="flex flex-col items-center p-4"
+        >
+          <icon-lucide-alert-triangle class="svg-icons mb-2 opacity-75" />
+          <h1 class="heading text-center">
+            {{ t("error.invalid_link") }}
+          </h1>
+          <p class="mt-2 text-center">
+            {{ t("error.invalid_embed_link") }}
+          </p>
+        </div>
+        <div
+          v-if="
+            !sharedRequestDetails.loading &&
```

Comment:
We can remove the `!sharedRequestDetails.loading` part of the check since it's already in the `else` block.

Reply:
Also, can you check if the `div` wrappers are necessary for the individual blocks? Since most of them share classes, we can move them to the common parent.

```vue
<div class="flex ... p-4">
  <HoppSmartSpinner ... />>
  
  <template v-else-if="E.isLeft(sharedRequestDetails.data)">
    <icon ...>
    <h1> ... </h1>
    <p> ... </p>
  </template>
  
    <template v-else-if="E.isRight(sharedRequestDetails.data)">
      <h1> ... </h1>
  </template>
</div>
```